pkg/kubelet/status/status_manager.go

```diff
@@ -186,6 +186,9 @@ func isPodStatusByKubeletEqual(oldStatus, status *v1.PodStatus) bool {
 }
 
 func (m *manager) Start() {
+	// Initialize m.state to no-op state checkpoint manager
+	m.state = state.NewNoopStateCheckpoint()
```

Comment:
nit: if we uncover more like this, it can be easier to put this in the else block below so when the feature graduates, we can be sure we just remove the else block.